WordPress/Classes/Utility/WebViewControllerFactory.swift

```diff
@@ -40,3 +40,12 @@ class WebViewControllerFactory: NSObject {
     }
 
 }
+
+extension WebViewControllerFactory {
```

Comment:
@momo-ozawa What do you think you can create a `JetpackControllerFactory`?

This is breaking the single responsibility principle, by making the `WebViewControllerFactory` specialized in creating a webview for this Jetpack URL.

Reply:
Fixed! 47a94e0